### PR TITLE
Add missing import for enter_userinfo()

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
Failing test: https://openqa.suse.de/tests/4009395#step/user_settings/3

### CODE REVIEW

File: tests/installation/user_settings.pm [hunk 1]
```diff
@@ -23,6 +23,7 @@ use parent qw(installation_user_settings y2_installbase);
 use testapi;
 use version_utils 'is_sle';
 use utils;
+use installation_user_settings;
```
Comment (alvarocarvajald) [inline tests/installation/user_settings.pm]: Won't importing `installation_user_settings` twice cause problems?

Comment (alvarocarvajald) [reply tests/installation/user_settings.pm]: Nevermind

**Comment by SergioAtSUSE** [comment]
Trying to verify: https://openqa.suse.de/tests/4013025

**Comment by SergioAtSUSE** [comment]
Trying with a different worker that is available: https://openqa.suse.de/tests/4013472

**Comment by SergioAtSUSE** [comment]
third try for victory?
https://openqa.suse.de/tests/4013473

**Comment by SergioAtSUSE** [comment]
Both jobs hang in welcome module :(

- https://openqa.suse.de/tests/4013544
- https://openqa.suse.de/tests/4013473

**Comment by alvarocarvajald** [comment]
> Both jobs hang in welcome module :(
> 
> * https://openqa.suse.de/tests/4013544
> * https://openqa.suse.de/tests/4013473

Just noticed that `installation/bootloader_zkvm` was not loaded.

**Comment by SergioAtSUSE** [comment]
> > Both jobs hang in welcome module :(
> > 
> > * https://openqa.suse.de/tests/4013544
> > * https://openqa.suse.de/tests/4013473
> 
> Just noticed that `installation/bootloader_zkvm` was not loaded.

:man_facepalming: 

nice catch

**Comment by SergioAtSUSE** [comment]
Trying on x86_64:

- http://openqa.slindomansilla-vm.qa.suse.de/tests/2329
- http://openqa.slindomansilla-vm.qa.suse.de/tests/2331

**Comment by foursixnine** [comment]
> Trying on x86_64:
> 
>     * http://openqa.slindomansilla-vm.qa.suse.de/tests/2329
> 
>     * http://openqa.slindomansilla-vm.qa.suse.de/tests/2331

Another http://phobos.suse.de/tests/3856939#live

**Comment by alvarocarvajald** [comment]
> > > Both jobs hang in welcome module :(
> > > 
> > > * https://openqa.suse.de/tests/4013544
> > > * https://openqa.suse.de/tests/4013473
> > 
> > 
> > Just noticed that `installation/bootloader_zkvm` was not loaded.
> 
> 
> 
> nice catch

AFAICS the problem could be due to this:

```
"INCLUDE_MODULES" : "welcome,accept_license,scc_registration,addon_products_sle,system_role,partitioning,partitioning_finish,installer_timezone,hostname_inst,user_settings",
```

However I don't think this needs to be tested in s390x. By sheer chance the test that failed was in that arch (all the other arches had already finished).

BTW, @foursixnine 's test already passed. I'd say we could merge.

**Comment by SergioAtSUSE** [comment]
I just rebased the fixup's.

When[if] the yast job passes, I put the PR in ready for review

**Pull request merged**